tests/features/warmshowers/not-logged-in.feature

```diff
@@ -1,40 +0,0 @@
-Feature: User profile should say not available when user marks status as unavailable
```

Comment:
Right this is the one we were discussing- its title is not logged in, but the content is about marking user as not available.  I think I've included all the content for this one in my set.